### PR TITLE
Add JSON directory listing for static sites

### PR DESCRIPTION
**Pull request by pixelblend**
The immediate use for this is to allow the music player to see what mp3
files are available to play.

### CODE REVIEW

File: services/manager/lib/io/http/index.js [hunk 1]
```diff
@@ -1,9 +1,18 @@
 const express = require('express');
 const ip = require('ip');
 const pathLib = require('path');
+const serveIndex = require('serve-index')
 
 const logger = require('../logger');
 
+const directoryListing = ({ fileList, directory }, callback) => {
+  const listing = fileList
+    .filter((f) => (f['name'][0] != '.'))
```
Comment (andrewn) [inline services/manager/lib/io/http/index.js]: The docs for `serve-index` makes it sound like it [shouldn't be listing hidden files](https://www.npmjs.com/package/serve-index#hidden). Is that not happening?

Comment (andrewn) [inline services/manager/lib/io/http/index.js]: You could also use `f.name`?

Comment (pixelblend) [reply services/manager/lib/io/http/index.js]: It includes the `..` path up to the parent directory

Comment (andrewn) [reply services/manager/lib/io/http/index.js]: To put it another way, Y U No right `f.name`?

**Pull request merged**